src/schema.rs

```diff
@@ -1202,9 +1199,9 @@ $ rust-cmd1
         let input = "One\nTwo\nThree";
         let line_nums = 2..3;
         let replacement = "World\n";
-        let expected = Data::text("One\nWorld\nThree");
+        let expected = "One\nWorld\nThree";
 
-        let mut actual = Data::text(input);
+        let mut actual = input.to_string();
```

Comment:
`to_string`
- Obscures intent
- Is sloppier, making it harder to safely refactor code since it will work with any `display`

In this case, I would use `.to_owned`